src/zcash/address/unified.cpp

```diff
@@ -122,17 +122,14 @@ UnifiedAddressGenerationResult ZcashdUnifiedFullViewingKey::FindAddress(
         const std::set<ReceiverType>& receiverTypes) const {
     diversifier_index_t j0(j);
     bool hasTransparent = HasTransparent(receiverTypes);
```

Comment:
`hasTransparent` is not used. Did you intend to delete the `hasTransparent && !j0.ToTransparentChildIndex().has_value()` error check?

Reply:
Yes, it was superfluous according to @nuttycom: https://github.com/zcash/zcash/pull/5469#discussion_r801145672

We can remove the unused line in a subsequent PR.

Reply:
The above link doesn't resolve because the comment is resolved. It's this one: https://github.com/zcash/zcash/pull/5469#pullrequestreview-875005920

Reply:
Ah I see: https://github.com/zcash/zcash/pull/5469#discussion_r801145672

```suggestion
```